tests/test.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
-import os from 'os';
-import fs from 'fs-extra';
-import path from 'path';
-import test from 'ava';
-import nock from 'nock';
-import sinon from 'sinon';
-import AWS from 'aws-sdk';
-import archiver from 'archiver';
-import { runTask, runServiceFromActivity } from '../index';
+const os = require('os');
```

Comment:
It doesn't matter, but why the change?

Reply:
Honestly, I'm not sure how this was working in the first place. When I upgraded to newer package versions, these tests started failing.